frontend/src/components/AppSidebar.tsx

```diff
@@ -181,19 +181,19 @@ export function AppSidebar() {
                   size="lg"
                   className="data-[state=open]:bg-sidebar-accent data-[state=open]:text-sidebar-accent-foreground"
                 >
-                  {info?.albyAccountConnected && (
```

Comment:
Not sure if this is good, or should we do `info?.albyAccountConnected` with ternary to avoid mix up